guide/sections/part2/wis2node.adoc

```diff
@@ -4,24 +4,33 @@
 
 ===== Registration and decommissioning of a WIS node
 
-This section describes the process used to register or remove a WIS node within WIS2. During the initial part of the WIS2 pilot phase, a Member simply needs to notify the WMO Secretariat and primary GISC of the intent to register a new WIS node. The Secretariat and GISC will then assist in the registration. More formal procedures will be developed as the number of WIS nodes increases.
+Registration and decomissionig of WIS nodes must be approved by the PR for the centre registering or decomissioning a WIS node.
 
-TODO: To be completed
+Procedure for PR Approved WIS node Registration is to create a Centre-ID based on naming convention ab-domain-nodename.  Where ab is the the IANA Top Level Domain code, domain is the organization's domain name for the main website, and nodename (optional) is descriptive of the purpose of the WIS node (for example a node for a type of data or programme of the data e.g. climate, aviation, etc.) when the organization is hosting multiple WIS nodes. A sample centre-ID is ab-myorg-climate.  In addition to the centre-ID the registering organization should also provide the broker endpoints.  Once centre-ID and broker endpoints are provided and entered into the WIS2 node database, the WMO Secretariat will contact the GISC for the organization's country and request the GISC to verify the correctness of the provided information.  The GISC will request the centre produce a test message with associated files for download.  The GISC should use a test MQTT client to verify the notification message is correct and the download links are functional.  The GISC notifies the Centre of the results of the checks and if all are good, requests the Centre to provide metadata when the dataset is ready.  When metadata is proivided the GISC informs the WMO Secretariat that the new centre-ID is ready and requests it be added to WIS2.
```

Comment:
"Where ab is the the IANA Top Level Domain code, domain is the organization's domain name for the main website, and nodename (optional) is descriptive of the purpose of the WIS node (for example a node for a type of data or programme of the data e.g. climate, aviation, etc.) when the organization is hosting multiple WIS nodes."

Information about the centre-id rules will be in an appendix of the Manual for Topic Hierarchy.... Maybe add a reference to that here instead?